src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
In Checklist:
#2 [CODE STYLE]: Use switch statement if you have limited amount of conditions.
Remake it with "swith  case" instead "if".